cmd/gctcli/data_history.go

```diff
@@ -0,0 +1,515 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/common/convert"
+	"github.com/thrasher-corp/gocryptotrader/currency"
+	"github.com/thrasher-corp/gocryptotrader/gctrpc"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	guidExample            = "deadbeef-dead-beef-dead-beef13371337"
+	specificJobSubCommands = []cli.Flag{
+		&cli.StringFlag{
+			Name:  "id",
+			Usage: guidExample,
+		},
+		&cli.StringFlag{
+			Name:  "nickname",
+			Usage: "binance-spot-btc-usdt-2019-trades",
+		},
+	}
+	fullJobSubCommands = []cli.Flag{
+		&cli.StringFlag{
+			Name:     "nickname",
+			Usage:    "binance-spot-btc-usdt-2019-trades",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:  "exchange",
+			Usage: "binance",
+		},
+		&cli.StringFlag{
+			Name:  "asset",
+			Usage: "spot",
+		},
+		&cli.StringFlag{
+			Name:  "pair",
+			Usage: "btc-usdt",
+		},
+		&cli.StringFlag{
+			Name:        "start_date",
+			Usage:       "formatted as: 2006-01-02 15:04:05",
+			Value:       time.Now().AddDate(-1, 0, 0).Format(common.SimpleTimeFormat),
+			Destination: &startTime,
+		},
+		&cli.StringFlag{
+			Name:        "end_date",
+			Usage:       "formatted as: 2006-01-02 15:04:05",
+			Value:       time.Now().AddDate(0, -1, 0).Format(common.SimpleTimeFormat),
+			Destination: &endTime,
+		},
+		&cli.Uint64Flag{
+			Name:  "interval",
+			Usage: klineMessage,
+		},
+		&cli.Uint64Flag{
+			Name:  "request_size_limit",
+			Usage: "500 - will only retrieve 500 candles per API request",
+		},
+		&cli.Uint64Flag{
+			Name:  "data_type",
+			Usage: "0 for candles, 1 for trades",
+		},
+		&cli.Uint64Flag{
+			Name:  "max_retry_attempts",
+			Usage: "3 - the maximum retry attempts for an interval period before giving up",
```

Comment:
You could set these values as per the ones you've suggested instead of in the usage blurb